e2e/nomostest/new.go

```diff
@@ -96,6 +96,18 @@ func newOptStruct(testName, tmpDir string, t nomostesting.NTB, ntOptions ...ntop
 		t.Skip("Test skipped for non-local GitProvider types")
 	}
 
+	if *e2e.TestCluster == e2e.GKE { // required env vars for GKE
+		if nomostesting.GCPProjectIDFromEnv == "" {
+			t.Fatal("Environment variable 'GCP_PROJECT' is required for GKE clusters")
+		}
+		if nomostesting.GCPClusterFromEnv == "" {
+			t.Fatal("Environment variable 'GCP_CLUSTER' is required for GKE clusters")
+		}
+		if nomostesting.GCPRegionFromEnv == "" && nomostesting.GCPZoneFromEnv == "" {
```

Comment:
Should we still error if both are set, like the shell script did?

Reply:
Yeah I think so, I'll update this

Reply:
Done